xbzu1/1.0/board.xml

```diff
@@ -60,37 +61,247 @@
    </jumpers>
 
    <components>   
-      <component 
-         name="part0" 
-         display_name="ZU+ MPSoC" 
-         type="fpga" 
-         part_name="xczu1cg-sbva484-1-e" 
-         pin_map_file="part0_pins.xml" 
-         vendor="xilinx" 
-         spec_url="http://avnet.me/XBoard-ZU1">
+
+      <!-- ZU+ MPSoC -->
+      <component name="part0" display_name="ZU+ MPSoC" type="fpga" part_name="xczu1cg-sbva484-1-e" pin_map_file="part0_pins.xml" vendor="xilinx" spec_url="http://avnet.me/XBoard-ZU1">
          <description>Zynq UltraScale+ part on the board</description>
          <interfaces>
-            <interface 
-               mode="master" 
-               name="ps8_fixedio" 
-               type="xilinx.com:zynq_ultra_ps_e:fixedio_rtl:1.0" 
-               of_component="ps8_fixedio" 
-               preset_proc="zynq_ultra_ps_e_preset"> 
+
+            <!-- ZU+ PS Fixed I/O -->
+            <interface mode="master" name="ps8_fixedio" type="xilinx.com:zynq_ultra_ps_e:fixedio_rtl:1.0" of_component="ps8_fixedio" preset_proc="zynq_ultra_ps_e_preset"> 
                <preferred_ips>
                   <preferred_ip vendor="xilinx.com" library="ip" name="zynq_ultra_ps_e" order="0"/>
                </preferred_ips>
             </interface>
-         </interfaces> 
```

Comment:
Removing this line looks like an error.  It will mean, in xml speak, the <interfaces> tag was "opened" (<>)but never "closed"(</>)

Reply:
It's still there but now shows up on line 214.